source/includes/steps-install.yaml

```diff
@@ -1,67 +1,82 @@
 title: "Install Xcode from the App Store."
 ref: install-xcode
 content: |
-  
-  Install Xcode. This installation may take awhile. In the meantime, you can follow the rest of the steps.
+
+  Install `XCode <https://developer.apple.com/xcode/download/>`__. This installation
+  may take some time.
+
+  If you are running macOS Mojave, install the macOS SDK headers once the XCode
+  installation has finished:
+
+  .. code-block:: sh
+
+     open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
 
 ---
 title: Download and install Homebrew
 ref: install-homebrew
 content: |
 
-  If homebrew is not installed, download it `here <https://brew.sh>`_.
+  If `Homebrew <https://brew.sh>`__ is not installed, install it:
 
-  Then run:
+  .. code-block:: sh
 
-  .. code-block:: sh 
-   
      /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-  
-  When this completes, run the following two commands:
+
+  When this completes, run the following commands:
 
   .. code-block:: sh
 
-     brew update 
+     brew update
      brew doctor
 
 ---
 title: Install Python 2 and Python 3
 ref: install-python
 content: |
-  
-  To run the build tools you need both python and python 2. Install them both by running:
+
+  To run the build tools you need both Python 3 and 2. Install them both by running:
```

Comment:
We might emphasize that you can't/shouldn't use built-in Python, lest it ruin your life.

Reply:
Updated!